client/transactions/list/index.tsx

```diff
@@ -221,13 +221,14 @@ export const TransactionsList = (
 				: [];
 		const riskLevel = <RiskLevel risk={ txn.risk_level } />;
 
-		const customerName = txn.order ? (
-			<Link href={ txn.order.customer_url ?? '' }>
-				{ txn.customer_name }
-			</Link>
-		) : (
-			txn.customer_name
-		);
+		const customerName =
+			txn.order && txn.order.customer_url ? (
+				<Link href={ txn.order.customer_url ?? '' }>
+					{ txn.customer_name }
+				</Link>
+			) : (
+				txn.customer_name
+			);
```

Comment:
@kalessil No need to reach for  `get_formatted_billing_full_name` and `get_billing_email` or other methods. The `customer_name` is already available on the frontend.

Reply:
@KarlisJ, awesome!